devicetypes/smartthings/zwave-multi-metering-switch.src/zwave-multi-metering-switch.groovy

```diff
@@ -55,32 +62,68 @@ metadata {
 		main(["switch"])
 		details(["switch","power","energy","refresh","reset"])
 	}
+	
```

Comment:
Since Your devices have to be integrated in the DTH that also supports other devices - the code for custom preferences cannot be supported here. Please remove all the code related to custom preferences.
If You want to configure Your device with fixed values, please use configure() method to send configuration commands.

Reply:
@KKlimczukS Have updated as requested.